packages/myst-frontmatter/src/frontmatter/frontmatter.spec.ts

```diff
@@ -47,25 +47,28 @@ const TEST_BIBLIO: Biblio = {
 };
 
 const TEST_THEBE: Thebe = {
-  useBinder: false,
-  useJupyterLite: false,
-  requestKernel: true,
-  binderOptions: {
-    binderUrl: 'https://my.binder.org/blah',
+  lite: false,
+  binder: {
+    url: 'https://my.binder.org/blah',
     ref: 'HEAD',
     repo: 'my-repo',
-    repoProvider: 'github',
+    provider: 'github',
   },
-  serverSettings: {
-    baseUrl: 'http://localhost:8888',
-    token: 'test-secret',
-    wsUrl: 'ws://localhost:8888',
-    appendToken: true,
+  server: {
+    baseUrl: 'https://my.server.org',
+    token: 'legit-secret',
   },
-  kernelOptions: { kernelName: 'python 3', name: 'python3', path: 'some-path' },
-  savedSessionOptions: { enabled: true, maxAge: 383000, storagePrefix: 'thebe' },
+  kernelName: 'python3',
```

Comment:
Should `kernelName` and `sessionName` be defined at the top level here or should the be nested under `server`?

Alternatively, do we even need the `server` key or should we lift `baseUrl` and `token` up a level?

I'm totally fine with this as-is, just asking questions...

Reply:
I defined them at top level as they can equally apply to the `binder` configuration, as well as the `server` configuration.
If `binder` is specified that will win over anything supplied in `server` (actually in `thebe` connecting to binder causes any `serverSettings` provided to be overridden`